contrib/node/tests/node/node-thrift-js-test/BUILD

```diff
@@ -0,0 +1,16 @@
+node_module(
+  name='node-thrift-js-test',
+  sources=globs('package.json', 'yarn.lock', '.babelrc', 'test/*.js'),
+  package_manager='yarn',
+  dependencies=[
+    'contrib/node/examples/src/node/employee-dep:employee-dep'
+  ],
+)
+
+# TODO(#8168): Make JS test compatible with newer Mocha versions.
```

Comment:
Ideally we are forward-moving here when adding new tests, but I also understand that your timeframe is somewhat limited here.